nemoguardrails/library/gotitai/flows.co

```diff
@@ -0,0 +1,7 @@
+define subflow gotitai rag truthcheck
+  """Guardrail based on the maximum risk score."""
+  $result = execute call gotitai truthchecker api
```

Comment:
The flow should follow the same pattern as for the existing hallucination rail, i.e., be conditioned on the `check_hallucination` context variable. This will enable the check to be enabled for certain types of questions and not for others: https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/nemoguardrails/library/hallucination/flows.co#L21

See this as well for more context: https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/docs/user_guides/guardrails-library.md#blocking-mode.

Reply:
We are reconsidering the hierarchy of rails in NeMo Guardrails and how we should position them. However, this will change for the April/May release. I suggest we merge this as a "fact-checking" method and refactor it with the rest of the code in 1-2 months.

Reply:
This is implemented in the latest commit